system-test/test.kitchen.ts

```diff
@@ -13,7 +14,7 @@
 // limitations under the License.
 
 import * as assert from 'assert';
-import {describe, it, before, after} from 'mocha';
+//import {describe, it, before, after} from 'mocha';
```

Comment:
The need for commenting this out was likely fixed by:

https://github.com/mochajs/mocha/pull/4599

```suggestion
import {describe, it, before, after} from 'mocha';
```